borg/localrole/workspace.py

```diff
@@ -103,20 +118,21 @@ def clra_cache_key(method, self, user, obj, object_roles):
 
           >>> original = rm._get_userfolder
           >>> def logger(self, *args, **kw):
-          ...     print 'checkLocalRolesAllowed called...'
+          ...     print('checkLocalRolesAllowed called...')
           ...     return original(self, *args, **kw)
           >>> rm._get_userfolder = logger
 
-          >>> print rm.checkLocalRolesAllowed(john, obj, ['foo', 'bar'])
+          >>> print(rm.checkLocalRolesAllowed(john, obj, ['foo', 'bar']))
           checkLocalRolesAllowed called...
           None
           >>> [i for i in IAnnotations(request)]
-          ["borg.localrole.workspace.checkLocalRolesAllowed:('john', '42!', ('foo', 'bar'))"]
+          ["borg.localrole.workspace.checkLocalRolesAllowed:
```

Comment:
this breaks the doctest: http://jenkins.plone.org/job/pull-request-5.0/334/testReport/junit/borg.localrole/workspace/clra_cache_key/